acceptance-tests/README.md

```diff
@@ -108,6 +123,38 @@ nightwatch src/test/js/
 Make sure you add the acceptance-tests to your intellij blueocean project by right-clicking on `acceptance-tests/pom.xml`
 and adding it to the project.
 
+### WebDriverMixin
+
+There is a `WebDriverMixin` that can help simplify writing tests significantly. It offers a number of
+utility methods for the most common operations, all which have logic added to deal with common problems
+in Blue Ocean such as animation.
+
+It is recommended you use this when writing tests.
+
+A simple example:
+
+```java
+import io.blueocean.ath.WebDriverMixin;
+
+public class LoginTest implements WebDriverMixin {
+    @Test
+    public void login() {
+        // go: goes to a relative jenkins path or absolute if starts with http(s)://
+        go("/login");
+        // find: waits for elements to be preset, if starts with / uses xpath, otherwise CSS selectors
+        find("#j_username") // find by ID when starts with #, this is just a CSS selector
+            // setText: clears, sets text on an input
+            .setText("admin");
+        find("input[name=j_password]") // just a CSS selector
+            .setText("admin");
+        // click: handles animation, failures will retry up to 2 times
+        click("//button[contains(text(), 'log')]"); // XPath selector
```

Comment:
I know this is a nit, but... could we revise this so that the xpath example is not giving an example of something we explicitly discourage (looking up a button based on its `text` content)? You should an xpath example below, so maybe this can just be a simple CSS selector.